src/main/scala/org/ergoplatform/nodeView/history/modifierprocessors/FullBlockPruningProcessor.scala

```diff
@@ -27,6 +28,14 @@ class FullBlockPruningProcessor(config: NodeConfigurationSettings) {
     isHeadersChainSynced && minimalFullBlockHeight <= height
   }
 
+  /** Nearest snapshot height node can use for fast syncing (penultimate snapshot).
```

Comment:
This is not clear, please add some description. 
Also comments formatting is different form out usual, please use
```
/**
  * Nearest snapshot height node can use for fast syncing (penultimate snapshot).
  *
  * @param height - height of last block
  */
```

Reply:
@oskin1, fixed.